configure.ac

```diff
@@ -117,6 +117,9 @@ dnl Libtool 2.2.6 requires: rm -f
 RM="$RM -f"
 
 PKG_PROG_PKG_CONFIG
+AS_IF([test -z "$PKG_CONFIG"],[
```

Comment:
the most portable way to test for an empty string is
```
test "x$PKG_CONFIG" = "x"
```

Reply:
Ah right. done